non-free.md

```diff
@@ -107,7 +107,7 @@
 - [JIRA](https://www.atlassian.com/software/jira) `⊘ Proprietary` - A professional and extensible issue tracker `Java`
 - [RhodeCode](https://rhodecode.com) `⊘ Proprietary` - On-premise Source Code Management for Mercurial, Git & Subversion. `Python`
 - [BitBucket Server](https://www.atlassian.com/software/bitbucket/server) `⊘ Proprietary` - An enterprise-level Git solution similar to GitLab `Java`
-
+- [Gitpod](https://www.gitpod.io/self-hosted) `⊘ Proprietary` - Ready-to-code dev environments, self-hosted on your infrastructure. `TypeScrip/Go`
```

Comment:
Add a link to the source code at the end: https://github.com/gitpod-io/self-hosted

Reply:
Thanks! @epsilon-0 I have made the change.